eventuals/scheduler.h

```diff
@@ -53,7 +53,7 @@ class Scheduler {
 
     Context(Context&& that) = delete;
 
-    virtual ~Context() {}
+    virtual ~Context() = default;
```

Comment:
Doesn't need to be `virtual` because it's `final` and doesn't `override` any base class destructor!